Job adjustments to Era and general review

BillBows opened the pull request.

Commit:
Adjusting general merit values to Era

- **HP/MP**
  - Max per item: 8
  - Max combo: 8
    - 1→2→3→4→5→5→5→5
- **Attributes**
  - Max per item: 5
  - Max combo: 5
    - 3→6→9→9→9
- **Defensive Skills**
  - Max per item: 4
  - Max Combo: 20 (includes Weapon Skills)
    - 1→2→3→3
- **Weapon Skills**
  - Max per item: 8
  - Max Combo: 20 (includes Defensive Skills)
    - 1→2→3→3→3→3→3→3
- **Magic Skills**
  - Max per item: 8
  - Max combo: 16
    - 1→2→3→3→3→3→3→3
- **Others**
  - Max per item: 4
  - Max combo: 8
    - 1→2→3→4

Commit:
Assigning merits to meritable traits (COR)

- Winning Streak
- Loaded Deck

Commit:
Adjusting JA recast timers (COR)

- Wild Card: 1 to 2h
- Snake Eye and Fold: 5 to 15 minutes

Commit:
Adjusting JA enmity values (COR)

- Light/Dark Shot: 0 to 1 CE

Commit:
Double-Up

- Removing the ability of Snake Eye to guarantee a XI roll by a chance when using Double-Up.
- Caping the roll value to 12 while removing the Double-Up effect regardless of Snake Eye status in case of a perfect or busted roll.
- Removing the extra effect from XI rolls where it would reset Phantom Roll recast.

Commit:
Quick Shot

- Changing how Shots effect bonuses are handled and tweaked values.

Commit:
Double-Up (Part 2)

- Removing the extra effect of a XI roll where the Phantom Roll recast would be reduced to 30 seconds while it lasts

Ranzera reviewed (approved):
Basically good. I'd set Geo and Cowbell to 0 though, for good measure.

Commit:
Adjusting Phantom Rolls to Era (COR)

- Drachen and Puppet Rolls to be discussed
- Using old DSP values for Ninja Roll values as there is a lack of resources for the Era
- Beast Roll mod (and all the others related to pets) not implemented
- Gallant Roll values adjusted but mod to be implemented (damage reflection)

Also commenting out as much as possible out of Era functionality

Commit:
Merge pull request #1 from BillBows/corsair

Corsair adjustments
